lib/spack/spack/cmd/buildcache.py

```diff
@@ -417,7 +417,7 @@ def preview_fn(args):
     for spec in specs:
         print("Relocatable nodes")
         print("--------------------------------")
-        print(spec.tree(status_fn=spack.relocate.is_relocatable))
+        print(spec.tree(status_fn=lambda s: True))
```

Comment:
Wondering if we can completely remove the:
```
spack buildcache preview command
```
It doesn't seem to be synced with other commands, and it's not env aware...

Reply:
Ok, this was introduced in #9199 We can safely remove it if we get rid of `--allow-root`.

Reply:
Then I'll just remove the output of the command that it's going to be removed since --allow-root is the default.